pkg/liqo-controller-manager/resourceoffer-controller/resourceoffer_controller.go

```diff
@@ -73,6 +73,7 @@ type ResourceOfferReconciler struct {
 //+kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=capsule.clastix.io,resources=tenants,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings,verbs=get;list;watch;create;update;patch;delete;deletecollection
+//+kubebuilder:rbac:groups="",resources=secrets,verbs=get;create;update
```

Comment:
Are these permissions required?

Reply:
Yes, it needs them to grant permissions to the virtualKubelet init container

Reply:
Forgot that the controller manager is the one giving the permissions to the virtual-kubelet